ansible/roles/kong-api/defaults/main.yml

```diff
@@ -6371,3 +6375,111 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+      
+  - name: startPdfProfanity
+    uris: "{{ content_validation_service_prefix }}/startPdfProfanity"
+    upstream_url: "{{ content_validation_service_url }}/contentValidation/v1/startPdfProfanity"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'dataCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+       
+  - name: getPdfProfanity
+    uris: "{{ content_validation_service_prefix }}/getPdfProfanity"
+    upstream_url: "{{ content_validation_service_url }}/contentValidation/v1/getPdfProfanity"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'dataAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+      
+  - name: getPdfProfanityForContent
+    uris: "{{ content_validation_service_prefix }}/getPdfProfanityForContent"
+    upstream_url: "{{ content_validation_service_url }}/contentValidation/v1/getPdfProfanityForContent"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'dataAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+      
+  - name: calculateScore
+    uris: "{{ scoring_engine_service_prefix }}/add"
+    upstream_url: "{{ scoring_engine_service_url }}/action/scoring/add"
```

Comment:
we can remove "action" from the url... 
use versions - so that it would follow the remaining API pattern
this applies the below APIs too...